Dockerfile

```diff
@@ -2,13 +2,13 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-FROM golang:1.9.0 as builder
+FROM golang:1.9.2-alpine3.6 as builder
 
 COPY . /go/src/github.com/tsuru/bs
 WORKDIR /go/src/github.com/tsuru/bs
-RUN make build
```

Comment:
I think we could also remove the build entry from the Makefile and also from the .travis.yaml file to avoid confusion since they're no longer needed.

Reply:
done